clients/vscode/src/Config.ts

```diff
@@ -73,6 +74,11 @@ export class Config extends EventEmitter {
     }
   }
 
+  get useVSCodeProxy(): boolean {
+    const advancedSettings = this.workspace.get("settings.advanced", {}) as AdvancedSettings;
+    return advancedSettings["useVSCodeProxy"] || true;
```

Comment:
```suggestion
    return advancedSettings["useVSCodeProxy"] ?? true;
```
`anything || true` will always return `true`.